tests/integration/tier1/single-yum-transaction/test_check_for_latest_packages.py

```diff
@@ -0,0 +1,36 @@
+from conftest import SYSTEM_RELEASE_ENV
+from envparse import env
+
+
+PACKAGES_TO_VERIFY_RHEL_7 = [("shim-x64", "shim-x64-15.6-3.el7_9.x86_64")]
+# Match the latest version used for conversion, currently Oracle Linux 8.7
+PACKAGES_TO_VERIFY_RHEL_8_X = [("shim-x64", "shim-x64-15.6-1.el8.x86_64")]
+# Mainly for CentOS Linux 8.5
+PACKAGES_TO_VERIFY_RHEL_8_5 = [("shim-x64", "shim-x64-15.4-2.el8_1.x86_64")]
+
```

Comment:
```suggestion
from conftest import SYSTEM_RELEASE_ENV
from envparse import env
import subprocess

COMMAND = "curl -s 'https://yum.oracle.com/repo/OracleLinux/OL8/baseos/latest/x86_64/index.html' | grep -oP 'getPackage/\\Kshim-x64-[\\w.-]+(\\.x86_64)' | tail -1"

LATEST_IN_OL_REPO = subprocess.check_output(COMMAND, shell=True, text=True).strip()

PACKAGES_TO_VERIFY_RHEL_7 = [("shim-x64", "shim-x64-15.6-3.el7_9.x86_64")]
# Match the latest version used for conversion, currently Oracle Linux 8.7
PACKAGES_TO_VERIFY_RHEL_8_X = [("shim-x64", LATEST_IN_OL_REPO)]
# Mainly for CentOS Linux 8.5
PACKAGES_TO_VERIFY_RHEL_8_5 = [("shim-x64", "shim-x64-15.4-2.el8_1.x86_64")]

```

Reply:
good point on that. I will apply the change once we have results from tier1

Reply:
Possibly `repoquery` would be better